uefi-raw/src/lib.rs

```diff
@@ -106,121 +107,190 @@ impl From<Boolean> for bool {
     }
 }
 
-/// An IPv4 internet protocol address.
-#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, Ord, PartialOrd, Hash)]
-#[repr(transparent)]
-pub struct Ipv4Address(pub [u8; 4]);
```

Comment:
I think we should keep the uefi-raw types `Ipv4Address` and `Ip6Address`. Note that the `core::net` types are *not* `repr(C)`, so it's not safe to rely on them having the same representation. See https://github.com/rust-lang/rust/pull/78802 for some background.

Reply:
Yeah, I think you are right! Would have been a nice simplification, however!